lib/platform/bitbucket/index.ts

```diff
@@ -712,16 +717,50 @@ export async function updatePr({
     )
   ).body;
 
-  await bitbucketHttp.putJson(
-    `/2.0/repositories/${config.repository}/pullrequests/${prNo}`,
-    {
-      body: {
-        title,
-        description: sanitize(description),
-        reviewers: pr.reviewers,
-      },
+  try {
+    await bitbucketHttp.putJson(
+      `/2.0/repositories/${config.repository}/pullrequests/${prNo}`,
+      {
+        body: {
+          title,
+          description: sanitize(description),
+          reviewers: pr.reviewers,
+        },
+      }
+    );
+  } catch (err) /* istanbul ignore next */ {
+    logger.warn(
```

Comment:
Can we check the response for the reviewer specific error message? So we only retry if it's because of disabled reviewers?

Reply:
We can.  The error response I've observed in our logs is along these lines

```
"response": {
    "statusCode": 400,
    "statusMessage": "Bad Request",
    "body": {
        "type": "error",
        "error": {
            "fields": {"reviewers": ["Malformed reviewers list"]},
            "message": "reviewers: Malformed reviewers list"
        }
    },
    ...
}
```

Reply:
Added a check for the `reviewers: Malformed reviewers list` bitbucket error